includes/class-sensei-course.php

```diff
@@ -2810,6 +2810,19 @@ public static function flush_rewrite_rules( $post_id ){
 
 	}
 
+	/**
+	 * Optionally return the full content on the single course pages
+	 * depending on the users course_single_content_display setting
+	 *
+	 * @deprecated since 1.11.0
```

Comment:
This won't make it into this release. We can put this change into 1.12.0 instead.